src/index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Consider renaming files from `index.md` to `README.md` for more natural rendering on Github.